taxonomies/food/categories.txt

```diff
@@ -53356,11 +53349,12 @@ agribalyse_proxy_food_name:fr: Boisson lactée, lait fermenté ou yaourt à boir
 intake24_category_code:en: YOGD
 intake24_category_code:fr: YOGD
 
-< en: Yogurt drinks
-en: Flavoured yogurt drinks, Yoghurt drinks (flavoured)
+< en: Yogurt drinks without sugar
+en: Flavoured yogurt drinks without sugar, Flavoured drinkable yogurts without sugar, Yoghurt drinks (flavoured without sugar)
+fr: Yaourts à boire aromatisés sans sucre
 nl: Yoghurtdranken met smaak
 
-< en:Flavoured yogurt drinks
+< en:Flavoured yogurt drinks with sugar
```

Comment:
Vanilla yogurt drinks could be without sugar

Reply:
I suggest to rename 'Vanilla yogurt drinks' to 'Vanilla yogurt drinks with sugar' since all products in it have sugar. What do you say @Freso and @stephanegigandet ?